content/api/commands/intercept.md

```diff
@@ -22,136 +22,64 @@ Unlike [cy.route()](/api/commands/route), `cy.intercept()`:
 - does not require calling [cy.server()](/api/commands/server) before use - in fact, `cy.server()` does not influence `cy.intercept()` at all.
 - does not have method set to `GET` by default, but intercepts `*` methods.
 
-## Usage
+## Syntax
 
-```ts
-cy.intercept(url, routeHandler?)
-cy.intercept(method, url, routeHandler?)
-cy.intercept(routeMatcher, routeHandler?)
+### Route-matching Only (No Stubbing or Intercepting)
+```js
+cy.intercept(url)
+cy.intercept(method, url)
+cy.intercept(routeMatcher)
+cy.intercept(url, routeMatcher)
+cy.intercept(method, url, routeMatcher)
```

Comment:
I need to verify that using both `method` and `url` with `routeMatcher` is valid.

Reply:
I wonder what happens if we specify `method` and `url` with `routeMatcher` that contains `method` and `url`.